source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml

```diff
@@ -35,7 +36,7 @@
           <span class="item">Proyectos:</span>
           <!--ko if: isEvaluationEditable() && (userView == 1 || userView == 3) -->
           <div class="editable-projects" data-bind="css: { edition: evaluation.project(), 'top-tip': evaluation.project()}, attr: {'data-tips': evaluation.project()}">
-            <input type="text" placeholder="Agregá los proyectos" data-bind="value: evaluation.project, event: {keyup: evaluation.onKeyUpProject, focusin: evaluation.onFocusInProject, blur: evaluation.onBlurProject}" /><span class="pencil"></span>
+            <textarea type="text" id="projects" placeholder="Agregá los proyectos" data-bind="value: evaluation.project, heightUpdater: evaluation.project, valueUpdate: 'input', event: {keyup: evaluation.onKeyUpProject, focusin: evaluation.onFocusInProject, blur: evaluation.onBlurProject}" ></textarea><span class="pencil"></span>
```

Comment:
Well, since the data can be modified i the view model I am not sure if it will work.